Datascience_With_Python/Machine Learning/Algorithms/Logistic Regression/README.md

```diff
@@ -40,15 +40,26 @@ In Logistic regression, sigmoid function which is an 'S' shaped graph is used fo
 Hypothesis is taken as follows:
 
 Z = WX + B
+
 This Z is passed into the sigmoid function
 
 hΘ(x) = sigmoid (Z)
+
 where sigmoid function is given by:
 
 sigmoid(t)=1/(1+e^-t)
+
+<p align="center">
+  <img src="Sigmoid Curve.png" width="350" title="sigmoid curve">
+</p>
 We then specify a decision boundary according to which the values are classified into particular classes. After this, using the predicted values we can give different classes to the target variable.
 
 ## Workflow of the algorithm
 
+<p align="center">
+  <img src="Code flowchart.jpeg" width="350" title="flowchart of code example">
+</p>
```

Comment:
Try dropping the images directly in the README file where they are required

Reply:
I have updated the README file, please confirm if the images are visible to you now?
Thank you!